### PR TITLE
Fix lint error on HouseholdRelationship JS and enforce linting rules in Travis

### PR DESCRIPTION
**Pull request by hamishtaplin**
### What is the context of this PR?

- Fixes issue JS linting issue
- Enforces linting of JS

### How to review 

1. `yarn lint` to confirm no errors

### CODE REVIEW

File: gulp/scripts.js [hunk 1]
```diff
@@ -63,9 +63,14 @@ export function copyScripts() {
     .pipe(browserSync.reload({stream: true}))
 }
 
-export function lint() {
-  gulp.src([paths.scripts.input, `!${paths.scripts.dir}vendor/**/*`, `!${paths.scripts.dir}polyfills.js`])
+export function lint(done) {
```
Comment (jonnyshaw89) [inline gulp/scripts.js:66]: I would be good if the build failed before the selenium tests run. The fail fast approach should also help with build times.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: gulp/scripts.js (hunk 1 above)
Comment (benjefferies) [reply gulp/scripts.js:66]: 👍

**Review by benjefferies — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
is this still 'needs-high-level-review'?

File: app/assets/js/app/modules/relationships.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const opts = {
 class HouseholdRelationship extends EventEmitter {
   constructor(item) {
     super()
+    var hello =''
```
Comment (hamishtaplin) [inline app/assets/js/app/modules/relationships.js]: Hello!

**Review by hamishtaplin — COMMENTED** [review]
Remove rogue `var`

File: app/assets/js/app/modules/relationships.js (hunk 2 above)
Comment (jonnyshaw89) [reply app/assets/js/app/modules/relationships.js]: At least it tested the fix, as travis failed.

**Pull request merged**